### PR TITLE
Update kubernetes-integration-install-configure.mdx

### PR DESCRIPTION
**Pull request by csongnr**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by x8a** [comment]
Hey, @rhetoric101! Could you have a look at this one once it's ready for review? Thanks!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 1]
```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
...
+
+### Navigating the Kubernetes Integration guided install [#kubernetes-install-navigation] 
 
 <Callout variant="important">
-Make sure you are using the supported Kubernetes versions. Please check our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
+Make sure you are using the supported Kubernetes versions and make sure to check out the preliminary notes for your managed services or platforms on our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
 </Callout>
 
+#### Configure the Kubernetes Integration [#configure-kubernetes-integration] 
+
```
Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:91]: This part is a little confusing to me. Goes straight from h4  to bolded headers. maybe some explanation of what is going to follow might help.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 2]
```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
...
+Cluster name is the name we will use to tag your Kubernetes data with so that you can filter for the data specific to the cluster you’re installing this integration in. This is important if you choose to connect multiple clusters to your New Relic account so choose a name that you’ll recognize! 
+
+**Namespace for the integration**
+
+Namespace for the integration is the namespace we will use to house the Kubernetes integration in your cluster. We recommend using the default namespace of ‘newrelic’. 
+
+
+#### Select the additional data you want to gather [#selecting-additional-data-kubernetes-integration] 
+
+##### Scrape Prometheus Endpoints [#scraping-prometheus-endpoints]
```
Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:107]: These h5s are pretty confusing to follow. Hard to tell if they're a subset of 'select additional data' header. Also, there's a couple h6s below; I assume these are subsets of the h5s but it's hard to say. I wonder if we might make a collapser or table. I find a lot of this organization hard to parse

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 3]
```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
...
+Allowlist_key  container_name
+Allowlist_key  namespace_name
+Allowlist_key  pod_name
+Allowlist_key  stream
+Allowlist_key  log
+```
+
+Conversely, users who prefer more robust data can opt to fully enrich their logs by adding in label and annotation data to their log data. 
+
+**Full Enrichment Log Data**
```
Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:195]: Just a note to remove title case uses in this doc

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 4]
```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
...
+
+Users who are already leveraging Pixie's Community Cloud can provide an API key to connect Pixie to New Relic. This approach will embed Pixie's live UI into their New Relic account for easy access (via Pixie's Live Debugging tool), as well as write Pixie data into New Relic through the New Relic OpenTelemetry endpoint. 
+
+<img
+  title="service graph in live debugger"
+  alt="service-graph"
+  src={pixieLiveDebugging}
+/>
+
+**Self-Hosted Pixie**
```
Comment (zuluecho9) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:269]: why do we use these bold formats instead of H2s or H3s or whatever?

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx]: @zuluecho9 I had changed this locally. Please check my latest commit.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 5]
```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
...
+
+Choose the New Relic account that you want your Kubernetes data written to. 
+
+**Cluster name** 
+
+Cluster name is the name we will use to tag your Kubernetes data with so that you can filter for the data specific to the cluster you’re installing this integration in. This is important if you choose to connect multiple clusters to your New Relic account so choose a name that you’ll recognize! 
+
+**Namespace for the integration**
+
+Namespace for the integration is the namespace we will use to house the Kubernetes integration in your cluster. We recommend using the default namespace of ‘newrelic’.
```
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx]: @zuluecho9 I had fixed this locally. Please see the latest commit.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx]: @zuluecho9 I had removed all these h5s locally. Now that I've pushed this up, please see the latest commit.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx (hunk 3 above)
Comment (rhetoric101) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx]: @zuluecho9 I had removed a lot of instances of title case locally. Please see the latest commit now that I've pushed that up.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx (hunk 4 above)
Comment (rhetoric101) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx]: @zuluecho9 In my local instance, I had moved these items into collapsers. Now that I've pushed this commit up, check the latest.
